### PR TITLE
Using hautelook/phpass version 0.3.4 for hashing purposes

### PR DESCRIPTION
**Pull request by jgrossi**
```
PHPUnit 4.2.2 by Sebastian Bergmann.

Configuration read from /home/vagrant/Github/corcel/phpunit.xml

...........................................

Time: 1.33 seconds, Memory: 8.00Mb

OK (43 tests, 130 assertions)
```

### CODE REVIEW

File: src/Password/PasswordService.php [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
-use Illuminate\Hashing\BcryptHasher;
 
 require_once(__DIR__ . "/../../vendor/autoload.php");
 
 class PasswordService
 {
     function __construct()
     {
-        $this->wp_hasher = new BcryptHasher();
+        $this->hasher = new PasswordHash(8, false);
```
Comment (rundef) [inline src/Password/PasswordService.php]: [2nd param should be true](https://github.com/WordPress/WordPress/blob/a311bea6d11bceabfff96ed03a9908ff7ba10505/wp-includes/pluggable.php#L2005-2015), to get the same behaviour as wp

Comment (jgrossi) [inline src/Password/PasswordService.php]: Ok, thanks! I'm gonna send another PR.

Comment (jgrossi) [inline src/Password/PasswordService.php]: Done #124
